### DISCUSSION TITLE
Connect Failed not same with APP and remoteExample

### DISCUSSION THREAD

**Discussion by tt2nn** [schwabe/ics-openvpn#1612] [Configuration problems]
Hi：
When i used OpenVPN Connected APP to connect vpn. get faile and disconnect, I get error log like:

LOG: Transport Error: mbed TLS: SSL read error : X509 - Certificate verification failed, e.g. CRL, CA or signature check failed
Setting LAST_EVENT=CERT_VERIFY_FAIL
EVENT: CERT_VERIFY_FAIL info='mbed TLS: SSL read error : X509 - Certificate verification failed, e.g. CRL, CA or signature check failed'
Broadcasting LAST_EVENT=CERT_VERIFY_FAIL
eceived EVENT=DISCONNECTED
-------------------------------------------------------------------
Then i used remoteExample to connect same vpn. always reconnecting,  I get error log like:

MANAGEMENT: >STATE:1662409833,WAIT,,,,,,
MANAGEMENT: >STATE:1662409833,AUTH,,,,,,
VERIFY ERROR: depth=1, error=certificate is not yet valid: CN=DO-VPN CA, serial=566640800809079469109513829525409701738862451438
OpenSSL: error:0A000086:SSL routines::certificate verify failed
TLS_ERROR: BIO read tls_read_plaintext error
TLS Error: TLS object -> incoming plaintext read error
TLS Error: TLS handshake failed
TCP/UDP: Closing socket
IGUSR1[soft,tls-error] received, process restarting
>STATE:1662409834,RECONNECTING,tls-error,,,,,
----------------------------------------------------
SO how can i get same status and log.
OS：Google Pixel 3a XL Android 12 API 32
Thanks.

**Comment by schwabe** [comment]
> VERIFY ERROR: depth=1, error=certificate is not yet valid: CN=DO-VPN CA, serial=566640800809079469109513829525409701738862451438

There is a problem with the certificate or some of your time on the phone or server
